src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -20,16 +26,10 @@ import {
 import {requestExternalStoragePermission} from '../../../utils/permission';
 import {setChannel} from '../../../context/actions/setChannel';
 import {setParticipants} from '../../../context/actions/groupChat';
-import {addMemberGroup, removeMemberGroup, leaveGroup} from '../../../service/chat';
-import UserSchema from '../../../database/schema/UserSchema';
-import ChannelListSchema from '../../../database/schema/ChannelListSchema';
-import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
-import useUserAuthHook from '../../../hooks/core/auth/useUserAuthHook';
-import useChatUtilsHook from '../../../hooks/core/chat/useChatUtilsHook';
 
 const useGroupInfo = (channelId = null) => {
   const navigation = useNavigation();
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useLocalDatabaseHook` now returns a new function `refreshWithId`. This seems to be a more specific version of the `refresh` function, allowing you to refresh a specific item by its ID. This could potentially improve performance by reducing unnecessary data fetching or re-rendering. However, it's important to ensure that this function is used correctly and that the ID provided always corresponds to an existing item in the database.



<!-- This is an auto-generated comment by OSS CodeRabbit -->